### PR TITLE
Fixed spelling error and update bitlink to match Seattle event.

### PR DESCRIPTION
**Pull request by bm2yogi**

### CODE REVIEW

File: _data/events/Bellevue.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "title": "Global Day of Code Retreat 2017 - Eastside",
-  "url": "http://bit.ly/gdcr17Eastside",
+  "url": "http://bit.ly/gdcr17eastside",
```
Comment (rradczewski) [inline _data/events/Bellevue.json]: Please add the resolved link here. We only use it in links, so it'd be nice for people to know where the link will lead them to.

**Comment by bm2yogi** [comment]
Requested changes have been made. :)

**Comment by rradczewski** [comment]
Can you please see to what happened to that branch? I'd appreciate if you could cherry-pick your commits ontop of `coderetreat/coderetreat.github.io master` This one has some commits from @dcarral in there Oo

**Comment by rradczewski** [comment]
Woops, messed a lot of stuff up, sorry. I still have your commits and will add them to the website, sorry for the confusion....

**Pull request closed without merging**